index.rst

```diff
@@ -15,5 +10,265 @@ Change Record
+the first 3 months of the cycle is loaded before the start of the cycle, and the remaining
+part of the plan covering the last 3 months is loaded before the 4th month starts. This
+allows for minor fine-tuning of the second half of the cycle without going through the CCB
+approval.
+
+Epics-based Long-term Planning
+------------------------------
+
+As explained above, epics are used for planning and executing work within a cycle.
+
+In addition to that, epics are also extremely valuable for longer-term planning at a fine-grain level.
+When details of work for a given planning package are known, they can and should be captured through
+epics. Such epics can be freely created and changed at any time without any approvals. They
+should, of course, fit within the scope and budget of related PP. They can be useful for
+bottom-up analysis and validation of resources needed to implement a given PP. This allows
+to do detailed planning in flexible and agile way, while ensuring the scope/cost/schedule is
+well controlled and managed.
+
+Level of Effort
+---------------
+
+Two general rules are applied for the DM Team labor when determining whether an activity should be
+treated as LOE or non-LOE:
+
+* All activities that have a clear deliverable are tracked through non-LOE. Examples of non-LOE
+  activities include writing a new piece of software, purchasing new equipment, or adding new
+  machines to the cluster.
+
+* Activities that can not be predicted, as well as short, recurring routine activities are considered LOE.
+  Examples of LOE activities include replacing faulty disk drive, cluster maintenance, or writing monthly
+  status report.
+
+For planning purposes, 70% of available time of each software developer is assumed to be available
+for "pure, uninterrupted coding", and the remaining 30% is considered as "overhead", and tracked
+explicitly as LOE.
+
+A small fraction of all DM labor is performed by Contractors. The contractor labor is managed as LOE,
+directly in dollars.
+
+People
+======
+
+* DM Project Manager responsible for overall plan, schedule and priorities.
+
+* Work divided into smaller pieces, through WBS.
+
+* Each piece has a dedicated technical manager.
+
+* Main responsibilities of technical managers:
+
+  * Assemble the team capable of delivering work scoped through the WBS on-time and within budget.
+    Provide daily technical management and leadership for the team, monitor and optimize team performance.
+
+  * Work closely with the DM Project Manager on defining short and long-term plan and schedule for
+    their teams. Direct the execution of their team's plan, ensuring the team delivers on-time and within budget.
+
+  * Report group's activities as required, including reporting to the Earned Value Management System (EVMS)
+    used by LSST, and providing input for monthly status reports.
+
+* In a sense, DM PM "owns" the overall baseline plan (PPs, milestones), and technical manager "own" epics.
```

Comment:
Don't TCAMs define PPs, with oversight from the PM? And TCAMs define epics with oversight from the PM? What's the distinction being made here?